app/scenes/Collection.js

```diff
@@ -188,26 +190,26 @@ class CollectionScene extends React.Component<Props> {
             {collection.isEmpty ? (
               <Centered column>
                 <HelpText>
-                  <strong>{collection.name}</strong> doesn’t contain any
-                  documents yet.
-                  <br />
-                  Get started by creating a new one!
+                  <Trans
+                    components={{ collectionName: collection.name }}
+                    defaults="<strong>{{ collectionName }}</strong> doesn’t contain any documents yet. <br /> Get started by creating a new one!"
+                  />
```

Comment:
If we wrap `Trans` around the components here the output in the translations will be nicer, see:

https://react.i18next.com/latest/trans-component

Reply:
fixed